From d25496300f695d55b64b242c2b3f6fcc8ffcdab9 Mon Sep 17 00:00:00 2001
From: Zygmunt Krynicki <zygmunt.krynicki@canonical.com>
Date: Thu, 8 Oct 2015 12:38:24 -0700
Subject: Add support for old unittest2 version (0.5.1)

 Upstream relies on more recent version of unittest2 that has support for the
 'assertRaisesRegex' method. Fortunately this method is a simple alias of
 'assertRaisesRegexp' (with extra 'p' at the end). Since the future of
 unittest2 in Debian is uncertain this intermediate trivial patch can be used
 to have working and tested version of Padme

Origin: upstream
Forwarded: not-needed
Last-Update: 2015-03-09

Patch-Name: 001-old-unittest2-support
---
 padme/tests.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/padme/tests.py b/padme/tests.py
index fc55507..7475e31 100644
--- a/padme/tests.py
+++ b/padme/tests.py
@@ -858,7 +858,7 @@ class proxy_as_function(unittest.TestCase):
     def test_context_manager_methods_v2(self):
         """ Verify that redirecting __exit__ passes right arguments. """
         exc = Exception("boom")
-        with self.assertRaisesRegex(Exception, "boom"):
+        with self.assertRaisesRegexp(Exception, "boom"):
             with self.proxy:
                 try:
                     raise exc
